.github/actions/upgrade-indexer/common.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+FILES_OLD="/usr/share/wazuh-indexer/plugins/opensearch-security/securityconfig"
+FILES_NEW="/etc/wazuh-indexer/opensearch-security"
+declare -A files_old
+declare -A files_new
+PACKAGE_NAME="$1"
+MAJOR_MINOR_RELEASE=$(($2))
+REFERENCE_VERSION=43
+
+EQUAL=true
+
+# Check the system to differ between DEB and RPM
+function check_system() {
+
+    if [ -n "$(command -v yum)" ]; then
+        sys_type="rpm"
+    elif [ -n "$(command -v apt-get)" ]; then
+        sys_type="deb"
+    else
+        echo "Error: could not detect the system"
+        exit 1
+    fi
+
+}
+
+# Checks the version of Wazuh with 4.3 version, where path is different.
+function check_version() {
+    if [ -z "$MAJOR_MINOR_RELEASE" ]; then
+        echo "Error: second argument expected"
+        exit 1
+    fi
+
+    if [ "$MAJOR_MINOR_RELEASE" -gt "$REFERENCE_VERSION" ]; then
+        # same path
+        FILES_OLD="$FILES_NEW"
```

Comment:
Use braces, you might find interesting this post https://stackoverflow.com/a/18136920/20563646

Despite the fact that the use of braces is only needed to resolve ambiguity in expressions, having the variable limited gives a better visual appearance and avoids possible future problems in case it is modified within the encapsulation with double quotes.

Reply:
Done.